nexus/src/app/disk.rs

```diff
@@ -54,6 +54,30 @@ impl super::Nexus {
                         ),
                     });
                 }
+
+                // Reject disks where the size isn't at least
+                // MIN_DISK_SIZE_BYTES
+                if params.size.to_bytes() < params::MIN_DISK_SIZE_BYTES as u64 {
+                    return Err(Error::InvalidValue {
+                        label: String::from("size"),
+                        message: String::from(
+                            "total size must be at least 1 GiB",
```

Comment:
The message is incorrect here, right? `MIN_DISK_SIZE_BYTES` would need to included in the message instead.

Reply:
Justin, I was curious about how to handle this one. I'll take care of that now. Thanks for your input.